src/py/mat3ra/made/tools/modify.py

```diff
@@ -462,3 +464,39 @@ def rotate_material(material: Material, axis: List[int], angle: float) -> Materi
     atoms.wrap()
 
     return Material(from_ase(atoms))
+
+
+def displace_interface(
+    interface: Material,
+    displacement: List[float],
+    label: InterfacePartsEnum = InterfacePartsEnum.FILM,
+    use_cartesian_coordinates=True,
+) -> Material:
+    """
+    Displace atoms in an interface along a certain direction.
+
+    Args:
+        interface (Material): The interface Material object.
+        displacement (List[float]): The displacement vector in angstroms or crystal coordinates.
+        label (InterfacePartsEnum): The label of the atoms to displace ("substrate" or "film").
+        use_cartesian_coordinates (bool): Whether to use cartesian coordinates.
+
+    Returns:
+        Material: The displaced material object.
+    """
+    new_material = interface.clone()
+    if use_cartesian_coordinates:
```

Comment:
We can also check whether it already is in cartesian or not to avoid unnecessary call to `to_cartesian`

Reply:
actually, the material method needs this check -- we already have this check for basis.to_cartesian() and basis.to_crystal()